sycl/test/on-device/host-interop-task/interop-task.cpp

```diff
@@ -118,15 +118,16 @@ void test2() {
   buffer<int, 1> Buffer1{BUFFER_SIZE};
   buffer<int, 1> Buffer2{BUFFER_SIZE};
 
+  auto Device = default_selector().select_device();
   // init the buffer with a'priori invalid data
   {
-    queue Q;
+    queue Q(context(Device), Device);
```

Comment:
Why is this necessary? SYCL spec says, that default constructor should construct a queue using `default_selector`. Is that behavior changed? Should `sycl::queue` docs be updated to reflect it then?

Reply:
I found a bug related to the interop handle and contexts unrelated to this change.  This test change preserves the original behavior.

Reply:
Ok